README.md

```diff
@@ -361,6 +361,13 @@ transloco-keys-manager find --add-missing-keys
 transloco-keys-manager find -a
 ```
 
+- `addMissingKeys`: Delete missing keys that were not found by the detective in existing translation (defaults to `false`)
+
+```
+transloco-keys-manager find --delete-missing-keys
+transloco-keys-manager find -d
+```
+
```

Comment:
I wasn't able to use this syntax. It seems like this needs to be used with `extract`, but then it wipes out all previous translations with dashes.